prime-router/assets/csv-download-site/working/index.html

```diff
@@ -201,7 +201,7 @@ <h2>Previous seven days</h2>
     <section class="usa-identifier__section usa-identifier__section--usagov" aria-label="U.S. government information and services">
       <div class="usa-identifier__container">
         <div class="usa-identifier__usagov-description">
-          <strong>PRIME Data Hub support</strong> &nbsp;(555) 867-5309 | email@email.com
+          <p>PRIME Data Hub support: email@email.com</p>
```

Comment:
Following along with Rick's comments [here](https://github.com/CDCgov/prime-data-hub/pull/242/files#r553570752) we should change this to `CDC Prime Data Hub`